src/main/java/org/broadinstitute/hellbender/engine/filters/ReadStrandFilter.java

```diff
@@ -6,7 +6,7 @@
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
 /**
- * Keep only reads whose strand is forward or reverse
+ * Keep only reads whose strand is as specified.
```

Comment:
We should then remove any mention of strandedness. Also, we should mention the default behavior. How about

> Keep only reads that align either rightward (not 0x10) or leftward (0x10), as specified. The filter's default keeps rightward aligning reads (not 0x10). 

Looks like this filter has changed from what we describe in <https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_gatk_engine_filters_ReadStrandFilter.php>. There, it seems the default keeps rightward alignments (same) and you can reverse this to keep leftward alignments by using the `--filterPositive` argument.

But now we have a `--keepReverse` argument that allows keeping leftward alignments. Good change.